kibbeh/src/ui/SearchBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { SmSolidSearch } from "../icons";
+
+export interface SearchBarProps
+  extends React.ComponentPropsWithoutRef<"input"> {
+  inputClassName?: string;
+}
+
+export const SearchBar: React.FC<SearchBarProps> = ({
+  className = "",
+  inputClassName = "",
+  ...props
+}) => {
+  return (
+    <div
+      className={`w-full bg-primary-700 text-primary-300 focus-within:text-primary-100 rounded-lg ${className}`}
+    >
+      <div className="h-full mx-4 flex items-center pointer-events-none">
+        <SmSolidSearch />
+      </div>
+      <input
+        className={`w-full py-2 bg-transparent text-primary-100 placeholder-primary-300 focus:outline-none ${inputClassName}`}
```

Comment:
Missed `pr-4` in the input

Reply:
I think it's not needed. There's no reason for the input not to expand to the full width.

Reply:
It is needed so the text doesn't reach the end of the wrapper.

<img width="1204" alt="Screenshot 2021-03-08 at 23 57 43" src="https://user-images.githubusercontent.com/57961822/110392970-1577e600-806a-11eb-895f-977a2ebff790.png">

vs

<img width="1210" alt="Screenshot 2021-03-08 at 23 58 17" src="https://user-images.githubusercontent.com/57961822/110393010-288ab600-806a-11eb-91bb-a8e8a1462693.png">